bundles/org.openhab.binding.freeboxos/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -199,6 +219,10 @@
 		<label>Wifi Enabled</label>
 		<description>Indicates whether the wifi network is enabled</description>
 		<category>Switch</category>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Enabled</tag>
```

Comment:
I am asking myself why it is Switch + Enabled fir this channel type while it is Switch + Mode for "DECT enabled" ?

Reply:
I think dect enabled should be control+enabled

Reply:
Generally we use Mode when it relates to the operating functionality of a device and where the more specific Power/Light/Enabled/OpenState properties cannot specifically be used. And/or use Mode for multi state conditions (e.g. Heat/Cool/Auto/Off) versus binary states.

Reply:
PS for simplicity I am creating a new PR to collect any post fix changes to the 1/9 .. 7/9 PRs after they all will have been merged; I will not release it until all the others are done..

Reply:
Yes

Reply:
> Could you create a draft to at least keep track of it?

#18910